app/com/arpnetworking/metrics/portal/alerts/impl/DailyPartitionCreator.java

```diff
@@ -230,7 +239,7 @@ private void tick() {
 
         LocalDate d = startDate;
         boolean allPartitionsExist = true;
-        while (!d.equals(endDate)) {
+        while (d.compareTo(endDate) <= 0) {
```

Comment:
I did this to match the semantics of `create_daily_partition` which includes both endpoints.